woocommerce/woocommerce
# Support free products in My Subscriptions

### Changes proposed in this Pull Request:

This PR is part of the My Subscriptions feature branch.

My Subscriptions include a combination of WCCOM and .org hosted products.
This PR ensures that .org product installed and active status is correct.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Purchase and install DePay (or another .org product with product keys)
2. [Open My Subscriptions](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions) and confirm it's showing as installed. (It won't show up as active till we push a change to WCCOM that automatically sets this type of subscription to active. It'll just appear as installed, as shown in this [example](https://github.com/woocommerce/woocommerce/pull/40820#pullrequestreview-1684467736).)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Correctly show .org products as installed and active in My Subscriptions.

</details>
